package.json

```diff
@@ -55,6 +56,7 @@
     "vue-test-utils-helpers": "git+https://github.com/AmpleOrganics/vue-test-utils-helpers.git",
     "vuetify-loader": "^1.4.3",
     "vuex-class": "^0.3.2",
+    "vuex-persistedstate": "^2.7.1",
```

Comment:
This package is no longer required.

Reply:
@kialj876 Given that your merge clobbered some code that was recently changed, would it make sense to re-merge? Or maybe go through your diffs and watch for things you didn't mean to change?

Reply:
Yeah I had a few rebase issues - I was going through that before running out of time on Tuesday